pkg/types/notification.go

```diff
@@ -23,18 +23,18 @@ import (
 	"github.com/Peripli/service-manager/pkg/util"
 )
 
-// OperationType is the notification type
-type OperationType string
+// NotificationOperation is the notification type
```

Comment:
Because it will conflict with the generated named constant for the `Notification` type:
https://github.com/Peripli/service-manager/blob/e51e45e72b67ca4cc4a4ea64f9657b6dcdff0740/pkg/types/notification_gen.go#L11